milestones/serializers.py

```diff
@@ -66,3 +69,23 @@ def deserialize_milestone(milestone_dict):
         namespace=milestone_dict.get('namespace', ''),
         description=milestone_dict.get('description', '')
     )
+
+
+def serialize_requirements(requirements):
+    """
+    Convert JSON serializable object to string
+    """
+    if requirements is not None:
+        requirements = json.dumps(requirements)
+    return requirements
+
+
+def deserialize_requirements(requirements):
```

Comment:
Based on feedback from @robrap, the requirements field will now contain a JSON string. The client of the milestones API will decide what that JSON looks like. The milestones API only validates that the object passed by the client is JSON serializable.